pcsx2/gui/GlobalCommands.cpp

```diff
@@ -247,10 +248,11 @@ namespace Implementations
 			return;
 		g_Conf->GSWindow.Zoom = zoom;
 		
-		if ( zoom == 0 ) 
-            Console.WriteLn(L"(GSwindow) Zoom: 0 (auto, no black bars)");
-		else 
-            Console.WriteLn(L"(GSwindow) Zoom: %f", zoom);
+		if ( zoom == 0 ) {
+			OSDlog( Color_StrongBlue, true, "(GSwindow) Zoom: 0 (auto, no black bars)");
+		} else {
+			OSDlog( Color_StrongBlue, true, "(GSwindow) Zoom: %f", zoom);
+		}
```

Comment:
I added them when OSD and Console logging were two separate API calls.  I can remove them now, since it's one call again.